src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,79 @@
+const expensesService = require('../services/expenses.service.js');
+const usersService = require('../services/users.service.js');
+
+const get = (req, res) => {
+  const { from, to, userId, categories } = req.query;
+  const filteredExpenses = expensesService.filterByQuery(
+    categories,
+    from,
+    to,
+    userId,
+  );
+
+  return res.status(200).json(filteredExpenses);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  const expensesId = expensesService.getById(id);
+
+  if (!expensesId || !id) {
```

Comment:
The condition `!id` is unnecessary because `id` is always present if this function is executed. You should only check for `!expensesId`.